sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -26,84 +27,37 @@ if [ "${PV%9999}" = "${PV}" ] ; then
 else
 	KEYWORDS=""
 fi
-IUSE="elibc_glibc elibc_musl +libcxxrt libunwind +static-libs test"
-REQUIRED_USE="libunwind? ( libcxxrt )"
+IUSE="elibc_glibc elibc_musl libunwind +static-libs"
```

Comment:
I think the location of its headers is too tricky to find. And given the options of libcxxrt and libc++abi, are there many people who would use libc++ with libsupc++? If so, I could definitely bring it back.